lib/utils/getoptions.js

```diff
@@ -38,10 +40,19 @@ module.exports = function cwdResolver( callOptions, cwd ) {
 
 	options.packages = path.resolve( cwd, options.packages );
 
+	/* istanbul ignore if */
 	if ( !Array.isArray( options.packagesPrefix ) ) {
 		options.packagesPrefix = [ options.packagesPrefix ];
 	}
 
+	// Check if under specified `cwd` path, the git repository exists.
+	// If so, find a branch name that the repository is checked out. See #103.
+	if ( fs.existsSync( path.join( cwd, '.git' ) ) ) {
```

Comment:
What will mgit do if there's no git repo under cwd? AFAIR – it will crash anyway. So, why do we handle this case here?

Reply:
Actually, I may be mistaken here – it will crash if there's no `mgit.json`. Is that correct?

Reply:
Mgit requires a config (`mgit.json`) to work but the config does not have to be inside a git repository. Does it have any sense? I don't know. If the feature requires reading anything from the repository located at the same level where `mgit.json` is saved and there is a 0.01% chance that `.git` directory won't exist, I accept the risk and prevent such a situation.